README.md

```diff
@@ -1001,24 +830,25 @@ The official ADH samples are divided in multiple categories depending on the sce
             </a>
           </td>
           <td align="middle">
-            <a
-              href="https://dev.azure.com/osieng/engineering/_build/latest?definitionId=4668&repoName=osisoft%2Fsample-adh-pi_to_adh_transfer_verification-powershell&branchName=main"
-            >
               <img
                 src="https://dev.azure.com/osieng/engineering/_apis/build/status/product-readiness/ADH/osisoft.sample-adh-pi_to_adh_transfer_verification-powershell?repoName=osisoft%2Fsample-adh-pi_to_adh_transfer_verification-powershell&branchName=main"
                 alt="Build Status"
               />
-            </a>
           </td>
         </tr>
       </table>
     </td>
   </tr>
 </table>
+</details>
 
+<br>
 ~~ These libraries are tested by being used in other apps.
```

Comment:
I think that this line could be removed now that we have the status badges for the library